src/ncp/ncp_base_mtd.cpp

```diff
@@ -207,11 +207,6 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_PHY_FREQ>(void)
     return mEncoder.WriteUint32(freq_khz);
 }
 
-template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_PHY_CHAN_SUPPORTED>(void)
-{
-    return EncodeChannelMask(otLinkGetPhySupportedChannelMask(mInstance));
```

Comment:
This does not seem correct to me? (not what I remember/expect from the earlier code).
The previous implementation should have been using the `otLinkGetSupportedChannelMask()`? (which maps to `Mac::GetSupportedChannelMask()`. The same way the setter is using `otLinkSetSupportedChannelMask`.

Reply:
Looks like it was changed by this commit (probably a mistake). 
https://github.com/openthread/openthread/commit/96725a985ed57a64721c61c4721e5792b44264c5#diff-523683496c0ec331f8caa4137d95295b

Reply:
Yes, I have fixed it within this PR.